Dockerfile

```diff
@@ -0,0 +1,12 @@
+# syntax=docker/dockerfile:1.4
+
+FROM ghcr.io/foundry-rs/foundry:nightly
```

Comment:
Not being familiar with the general stability of this repository, I would bias towards `v1.0.0` (published 25 days ago) rather than rely on a version that is potentially broken (i.e. the CLI args we rely on for the deploy script change).

Prevents the "works on my machine" class of problems.